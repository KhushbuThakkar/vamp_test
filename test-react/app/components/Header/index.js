import React from 'react';
import { FormattedMessage } from 'react-intl';
import { Link } from 'react-router-dom';
import A from './A';
import Img from './Img';
import NavBar from './NavBar';
import HeaderLink from './HeaderLink';
import Banner from './banner.jpg';
import messages from './messages';
import Button from '../Button';

function Header() {
  return (
    <div>
      {/* <A href="https://www.reactboilerplate.com/">
        <Img src={Banner} alt="react-boilerplate - Logo" />
      </A> */}
      <NavBar>
        <Button as={Link} to="/" variant="primary" mr={2}>
          <FormattedMessage {...messages.home} />
        </Button>
        <Button as={Link} to="/features" variant="secondary">
          <FormattedMessage {...messages.features} />
        </Button>
      </NavBar>
    </div>
  );
}

export default Header;
