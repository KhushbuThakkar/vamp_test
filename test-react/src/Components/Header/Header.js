import React from "react";
import { Text } from "rebass";
import PropTypes from "prop-types";

import Box from "../Box/Box";

const Header = (props) => (
  <Box
    p={20}
    mb={10}
    sx={{
      mx: "auto",
      px: 3,
      bg: "primary",
    }}
  >
    <Text color="white">{props.text}</Text>
  </Box>
);

Header.propTypes = {
  text: PropTypes.string,
};

Header.defaultProps = {
  text: "",
};

export default Header;
