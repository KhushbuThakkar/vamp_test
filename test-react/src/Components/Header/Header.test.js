import React from "react";
import renderer from "react-test-renderer";
import Header from "./Header";

it("render correctly Header component", () => {
  const component = renderer.create(<Header text="Test Header" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
