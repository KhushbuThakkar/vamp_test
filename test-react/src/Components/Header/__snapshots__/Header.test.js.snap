// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Header component 1`] = `
<div
  className="css-1z0r4uk"
>
  <div
    className="css-zdzv7c"
  >
    Test Header
  </div>
</div>
`;
